app/services/phone_number_capabilities.rb

```diff
@@ -10,12 +10,24 @@ def self.load_config
   attr_reader :phone, :phone_confirmed
 
   def self.translated_international_codes
-    translated_international_codes_data = {}
-    INTERNATIONAL_CODES.each do |k, value|
-      translated_international_codes_data[k] =
-        value.merge('name' => I18n.t("countries.#{k.downcase}"))
+    if Rails.env.development?
```

Comment:
I'd wonder if it's even worth bothering with this development-mode handling. I'd be shocked if this caching ever turned out to be a problem.

(To future `git blame`-ers, I apologize if it turned out to be a problem)

Reply:
Alternatively, could we just add a line like this, which would force the initialization each time?

```rb
remove_instance_variable(:@translated_intl_codes_data) if Rails.env.development?
```

Reply:
I agree, different develpment branch of code seems like more of a maintenance burden, I'd see if we can share code

Reply:
There's some [precedent](https://github.com/18F/identity-idp/search?q=Rails.env.development%3F) within our code  (and I ran into a hidden one while trying to profile with https://github.com/heartcombo/simple_form/blob/86429bceb950096df3c29616f31bd5a5ce706c06/lib/simple_form.rb#L175)

It's going to be inconsistent in some form along either path (either by not changing content when certain YML files are updated or having dev behave differently than other environments).  Another option is configuration, but it doesn't feel like a good use-case for that.

I lean towards

```ruby
remove_instance_variable(:@translated_intl_codes_data) if Rails.env.development?
```

since it's much simpler, but I don't feel incredibly strongly

Reply:
Also, if it's always going to be a hash, could you just do a simple truthy check instead of `defined?` ?

```rb
def self.translated_international_codes
  @translated_intl_codes_data = nil if Rails.env.development?
  return @translated_intl_codes_data[I18n.locale] if defined?(@translated_intl_codes_data)
  @translated_intl_codes_data = Hash.new { |h, k| h[k] = {} }
  # ...
end
```

Reply:
Ah I like that, so I've pushed up a commit to do it.